api/config/module/feedback/v1/feedback.proto

```diff
@@ -20,14 +20,16 @@ message Survey {
   RatingOptions rating_options = 3 [ (validate.rules).message = {required : true} ];
 }
 
-message SurveyOrigin {
-  oneof type {
-    option (validate.required) = true;
+enum Origin {
+  // currently these are the supported placements for feedback
+  ORIGIN_UNSPECIFIED = 0;
+  HEADER = 1;
+  WIZARD = 2;
+}
 
-    // currently these are the supported origins for feedback
-    Survey header = 1 [ (validate.rules).message = {required : true} ];
-    Survey wizard = 2 [ (validate.rules).message = {required : true} ];
```

Comment:
changed it up a bit b/c in creating the survey lookup map, realized with this pattern i'd either have to do case switches on the key type, which would mean remembering to update `NewSurveyLookup` when a new placement is added. I could also have done a reflect.MapKeys() instead but with the new approach felt cleaner